### ISSUE TITLE
Uncompleted required tasks don't block update of a published post

### ISSUE THREAD

**Issue by andergmartins** [publishpress/PublishPress-Checklists#133]
In the Classic Editor if you edit a published post introducing something that breaks any completed task (example remove some chars, remove featured image, etc) you can update the post with no warning or without being blocked by the uncompleted tasks.

**Comment by andergmartins** [comment]
After the commit above it is fixed for the classic editor. For the gutenberg editor we still need to find a better way, because the undone tasks are displayed in the pre-publishing panel, which is not displayed or used when the post is already published. So when we update, the verification is not triggered.

@stevejburge maybe we should use a different approach for Gutenberg? In the classic editor we have 2 types of popups. One for blocking saving and displaying the required tasks. An another for displaying just a warning when there are only recommended tasks.
What about use a popup on Gutenberg for blocking the publish button and displaying the required tasks, and only show the warning in the pre-publishing panel if we have only recommended tasks?

**Comment by stevejburge** [comment]
@andergmartins Good question. Within the next 6 to 12 months, we can probably drop Classic Editor support and focus on Gutenberg.

In the short term, maybe you're right. Maybe a modal is the best single solution .... Classic Editor, Gutenberg, before publishing, after publishing etc

**Comment by andergmartins** [comment]
@stevejburge thanks. 

Yeah, it would be great to focus only on Gutenberg. But one problem on dropping support for Classic Editor is custom post types. They usually use the Classic Editor as base.

**Comment by stevejburge** [comment]
Good point. OK, modal for everything sounds like a good approach

**Comment by andergmartins** [comment]
We got today another request for fixing this

**Comment by stevejburge** [comment]
Closing in favor of https://github.com/publishpress/PublishPress-Checklists/issues/108
